news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -367,11 +368,11 @@ protected function generateLink($strLink, $objArticle, $blnAddArchive=false, $bl
 		$strArticleUrl = News::generateNewsUrl($objArticle, $blnAddArchive);
 
 		return sprintf(
-			'<a href="%s" title="%s"%s itemprop="url">%s%s</a>',
+			'<a href="%s" title="%s"%s>%s%s</a>',
 			$strArticleUrl,
 			StringUtil::specialchars(sprintf($strReadMore, $blnIsInternal ? $objArticle->headline : $strArticleUrl), true),
 			($objArticle->target && !$blnIsInternal ? ' target="_blank" rel="noreferrer noopener"' : ''),
-			($blnIsReadMore ? $strLink : '<span itemprop="headline">' . $strLink . '</span>'),
+			($blnIsReadMore ? $strLink : '<span class="headline">' . $strLink . '</span>'),
```

Comment:
```suggestion
			$strLink,
```

Reply:
I thought people were maybe using `itemprop="headline"` as a CSS selector so I did not want to take away the possibility to still select that.